Formula/stellar-core.rb

```diff
@@ -0,0 +1,29 @@
+class StellarCore < Formula
+  desc "The backbone of the Stellar (XLM) network"
+  homepage "https://www.stellar.org/"
+  url "https://github.com/stellar/stellar-core.git",
```

Comment:
We would prefer to build from a tarball (`https://github.com/stellar/stellar-core/archive/v9.1.0.tar.gz`) as this makes full use of Homebrew's caching mechanism.

Reply:
Looks like there's submodules, which might not be part of the standard `archive` tarball? Can't remember off the top of my head 😓.

Reply:
No, those folders are empty in the archive tarball. So would either need the `git clone` or a fuller release tarball, I guess.

Reply:
Yes unfortunately doesn't appear possible to build from tarball at the moment. That is what I had attempted first. For the moment, this works and is not the only formula resorting to git. I'll also do a follow up pull request to improve this as soon as the situation changes.